### DISCUSSION TITLE
How to claim for a username / organization which appears taken but it is not even available anywhere

### DISCUSSION THREAD

**Discussion by inglesuniversal** [community/community#29459] [Other Feature Feedback, Questions, & Ideas]
Is there a procedure to follow? I have been trying to register a new account under a new user or organization but it does not appear anywhere on the internet as even active, it just shows taken.

I have documentation to back up our request as legit owners of that particular name, created over 5 years ago.

Any ideas of whom to contact or how this process goes?

Thanks

**Comment by radar-blip** [comment]
If you're trying to claim a username thats already taken, theres not a whole lot you can do about it.
If it previously belonged to you, maybe. You would have to contact GitHub staff and make a request.
Here is a link that might help you: https://support.github.com/request

**Comment by rilysh** [comment]
GitHub doesn't allow dormant neither previous claimed usernames which account supposed to now deleted either inactive for a long while.

**Reply by voldemort8h** [reply]
for how long "while"? exactly

**Reply by inglesuniversal** [reply]
Somebody should recognize the need not to allow this to be happening, it is an utterly waste of time.

**Reply by queenofcorgis** [reply]
Hi @inglesuniversal,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!

**Comment by fredflev** [comment]
@queenofcorgis 

hello,

Same question here, two users have accounts that have been inactive for over many years but they are using my desired org/company name. 

Is there any chance the users can be contacted and the account released after no response from their side? 

One user created an account in 2016 and did not do anything with it. The other users created his account in 2015, made a single commit and did no do anything else since then. 

I would happily upgrade to a paid GitHub account if it can accelerate the resolution. 

Please let me know. 

Thanks.

Ps: I've read https://docs.github.com/en/site-policy/content-removal-policies/github-trademark-policy

**Reply by CynthiaWahome** [reply]
hi @fredflev 
any luck resolving this?

**Comment by matthias** [comment]
Any way to contact the owner of an inactive account directly. When they do not have any projects?

**Reply by barrettruth** [reply]
Same. I want `github.com/barrett` - the user's only activity is joining github 15 years ago! :cry:

**Comment by Anon-contribution** [comment]
Would be cool to have a feature for gihtub to send a message to the organization on behalf of another user.

**Comment by RudrakshKashyap** [comment]
@mhmy you are using my org name :/
